app/views/annotations/_form.html.erb

```diff
@@ -0,0 +1,151 @@
+            <span class="annotation_color<%=i%>"></span>
+          <% end %>
+      <% end %>
+    </div>
+    <!--- SIMPLE COLOR PICKER
+    <b>Color: &nbsp</b>
+    <%= f.color_field :color %> -->
+
+    <br>
+
+    <!--- CATEGORY RADIO BUTTON -->
+    <b><%= t('admin.annotation.category') %></b>
+    <div class="form-row ml-2", id="category-radios">
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :note,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.note'),
+                    value: :note,
+                    class: 'form-check-label' %>
+      </div>
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :content,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.content'),
+                    value: :content,
+                    class: 'form-check-label' %>
+      </div>
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :mistake,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.mistake'),
+                    value: :mistake,
+                    class: 'form-check-label' %>
+      </div>
+      <div class="form-check form-check-inline">
+        <%= f.radio_button :category,
+                           :presentation,
+                           class: 'form-check-input' %>
+        <%= f.label :category,
+                    t('admin.annotation.presentation'),
+                    value: :presentation,
+                    class: 'form-check-label' %>
+      </div>
+    </div>
+
+    <br>
+
+    <div id="specific">
+      <!--- SPECIFIC CONTENT, DEPENDING ON THE SELECTED CATEGORY, IS INSERTED HERE -->
+    </div>
+
+    <br>
+
+    <!--- VISIBLE FOR TEACHER? - CHECKBOX -->
+    <% annotations_status = Medium.find_by(id: @medium_id).get_annotations_status %>
+    <% if annotations_status == 1 %>
+      <%= f.label :visible_for_teacher, class: "checkbox inline" do %>
+        <span, title="<%= t('admin.annotation.visible_for_teacher_tooltip') %>"><b>
+          <%= t('admin.annotation.visible_for_teacher') %> &nbsp
+        </b></span>
+        <%= f.check_box :visible_for_teacher %>
+      <% end %>
+      <br>
+    <% elsif annotations_status == -1 %>
+      <%= f.hidden_field :visible_for_teacher, value: false %>
+    <% end %>
+
+    <%= f.label :visible_for_teacher, class: "checkbox inline" do %>
+      <span><b><%= t('admin.annotation.post_as_comment') %> &nbsp</b></span>
+      <%= f.check_box :post_as_comment %>
+    <% end %>
+
+    <br>
+
+    <div class="modal-footer">
+      <%= f.submit t('buttons.save'), class: "btn btn-primary", id: "submit-button" %>
```

Comment:
choose more specific id since `submit-button` is probably not unique (many components have a submit button)